extendable_fastapi/__manifest__.py

```diff
@@ -16,6 +16,7 @@
     "external_dependencies": {
         "python": [
             "extendable-pydantic>=0.0.4",
+            "fastapi<=0.108.0",
```

Comment:
Why do you pin the fastapi version?

Reply:
Actually I've done some more research and I've found out that [Starlette Version 0.35.0](https://github.com/encode/starlette/releases/tag/0.35.0) has some breaking changes that makes the tests fail.

I've previously pinned fastapi and thought it was working because `fastapi==0.108.0` has `starlette<0.33.0`

Reply:
@renda-dev Thank you for all your investigation! Could you explain what's wrong with starlette 0.35?

Reply:
I've digged some more and found out that they changed the way that middlewares are declared and loaded, now it should be fixed with the latest commit

Reply:
Thank you!